src/tools/sfdk/dispatch.cpp

```diff
@@ -253,11 +261,29 @@ void Dispatcher::registerWorkerType(const QString &name, const WorkerCreator &cr
     m_workerCreators.insert(name, creator);
 }
 
+QString Dispatcher::brandedString(const QString &value)
+{
+    if (value.isEmpty())
+        return {};
```

Comment:
That's a bit odd: if value is empty, return a null value :) I would simply check with isNull here as it is done in the other case.

Reply:
Fixed